.gitignore

```diff
@@ -57,6 +57,10 @@ distribute-*.tar.gz
 .tox
 .*.sw[op]
 *~
+
+# emacs save files
```

Comment:
This section broke up the existing `# Other` section. Would you move it before `# Other`?